### PR TITLE
Introduce StringBuilder.appendLine in stdlib-common #KT-37839

### PR DESCRIPTION
**Pull request by qurbonzoda**

### CODE REVIEW

File: runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt [hunk 1]
```diff
@@ -5,14 +5,47 @@
...
+fun StringBuilder.appendLine(value: Float): StringBuilder = append(value).appendLine()
+
+@Deprecated("Use appendLine instead", ReplaceWith("appendLine"), level = DeprecationLevel.WARNING)
+fun StringBuilder.appendln(it: Double) = appendLine(it)
+
+/** Appends [value] to this [StringBuilder], followed by a linefeed character (`\n`). */
+fun StringBuilder.appendLine(value: Double): StringBuilder = append(value).appendLine()
+
+@Deprecated("Use appendLine instead", ReplaceWith("appendLine"), level = DeprecationLevel.WARNING)
+fun StringBuilder.appendln(it: Any?) = appendLine(it)
```
Comment (ilya-g) [inline runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt:51]: please specify explicit visibility and return types

Comment (qurbonzoda) [reply runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt]: done

File: runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt [hunk 2]
```diff
@@ -5,14 +5,60 @@
...
+@kotlin.internal.InlineOnly
+public inline fun StringBuilder.appendLine(value: Float): StringBuilder = append(value).appendLine()
+
+/** Appends [value] to this [StringBuilder], followed by a line feed character (`\n`). */
+@SinceKotlin("1.4")
+@kotlin.internal.InlineOnly
+public inline fun StringBuilder.appendLine(value: Double): StringBuilder = append(value).appendLine()
+
+
+@Deprecated("Use appendLine instead", ReplaceWith("appendLine"), level = DeprecationLevel.WARNING)
```
Comment (ilya-g) [inline runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt:39]: Also replacement expression.

File: runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt [hunk 3]
```diff
@@ -5,14 +5,60 @@
...
-fun StringBuilder.appendln(it: Boolean) = append(it).appendln()
-fun StringBuilder.appendln(it: Byte) = append(it).appendln()
-fun StringBuilder.appendln(it: Short) = append(it).appendln()
-fun StringBuilder.appendln(it: Int) = append(it).appendln()
-fun StringBuilder.appendln(it: Long) = append(it).appendln()
-fun StringBuilder.appendln(it: Float) = append(it).appendln()
-fun StringBuilder.appendln(it: Double) = append(it).appendln()
-fun StringBuilder.appendln(it: Any?) = append(it).appendln()
-
-fun StringBuilder.appendln() = append('\n')
```
Comment (qurbonzoda) [inline runtime/src/main/kotlin/kotlin/text/StringBuilderNative.kt:18]: This method is missing

**Pull request merged**